Compose Preview does not work in modules under Feature.

**Idea Description**

- As shown in the attached screenshot, Preview works in modules under Core, but does not work in modules under Feature.
- Please make Preview work in Feature as well as in Core.

**Reference images and links**

<img width="500" height="1330" alt="Image" src="https://github.com/user-attachments/assets/3592c339-7bbc-493c-a854-e5893d039b39" />

<img width="500" height="1330" alt="Image" src="https://github.com/user-attachments/assets/0d5aaaac-cc90-4dd0-ad3e-84c6a698208b" />

```
java.lang.NoClassDefFoundError: androidx/compose/runtime/tooling/SourceInformationKt
	at androidx.compose.ui.tooling.data.SlotTreeKt.sourceInformationContextOf(SlotTree.jvm.kt:226)
	at androidx.compose.ui.tooling.data.SlotTreeKt.getGroup(SlotTree.jvm.kt:249)
	at androidx.compose.ui.tooling.data.SlotTreeKt.getGroup(SlotTree.jvm.kt:254)
	at androidx.compose.ui.tooling.data.SlotTreeKt.getGroup(SlotTree.jvm.kt:254)
	at androidx.compose.ui.tooling.data.SlotTreeKt.getGroup(SlotTree.jvm.kt:254)
	at androidx.compose.ui.tooling.data.SlotTreeKt.getGroup(SlotTree.jvm.kt:254)
	at androidx.compose.ui.tooling.data.SlotTreeKt.getGroup(SlotTree.jvm.kt:254)
	at androidx.compose.ui.tooling.data.SlotTreeKt.getGroup(SlotTree.jvm.kt:254)
	at androidx.compose.ui.tooling.data.SlotTreeKt.getGroup(SlotTree.jvm.kt:254)
	at androidx.compose.ui.tooling.data.SlotTreeKt.getGroup(SlotTree.jvm.kt:254)
	at androidx.compose.ui.tooling.data.SlotTreeKt.getGroup(SlotTree.jvm.kt:254)
	at androidx.compose.ui.tooling.data.SlotTreeKt.getGroup(SlotTree.jvm.kt:254)
	at androidx.compose.ui.tooling.data.SlotTreeKt.getGroup(SlotTree.jvm.kt:254)
	at androidx.compose.ui.tooling.data.SlotTreeKt.getGroup(SlotTree.jvm.kt:254)
	at androidx.compose.ui.tooling.data.SlotTreeKt.getGroup(SlotTree.jvm.kt:254)
	at androidx.compose.ui.tooling.data.SlotTreeKt.getGroup(SlotTree.jvm.kt:254)
	at androidx.compose.ui.tooling.data.SlotTreeKt.getGroup(SlotTree.jvm.kt:254)
	at androidx.compose.ui.tooling.data.SlotTreeKt.getGroup(SlotTree.jvm.kt:254)
	at androidx.compose.ui.tooling.data.SlotTreeKt.getGroup(SlotTree.jvm.kt:254)
	at androidx.compose.ui.tooling.data.SlotTreeKt.getGroup(SlotTree.jvm.kt:254)
	at androidx.compose.ui.tooling.data.SlotTreeKt.getGroup(SlotTree.jvm.kt:254)
	at androidx.compose.ui.tooling.data.SlotTreeKt.getGroup(SlotTree.jvm.kt:254)
	at androidx.compose.ui.tooling.data.SlotTreeKt.getGroup(SlotTree.jvm.kt:254)
	at androidx.compose.ui.tooling.data.SlotTreeKt.getGroup(SlotTree.jvm.kt:254)
	at androidx.compose.ui.tooling.data.SlotTreeKt.getGroup(SlotTree.jvm.kt:254)
	at androidx.compose.ui.tooling.data.SlotTreeKt.getGroup(SlotTree.jvm.kt:254)
	at androidx.compose.ui.tooling.data.SlotTreeKt.getGroup(SlotTree.jvm.kt:254)
	at androidx.compose.ui.tooling.data.SlotTreeKt.getGroup(SlotTree.jvm.kt:254)
	at androidx.compose.ui.tooling.data.SlotTreeKt.getGroup(SlotTree.jvm.kt:254)
	at androidx.compose.ui.tooling.data.SlotTreeKt.asTree(SlotTree.jvm.kt:480)
	at androidx.compose.ui.tooling.ComposeViewAdapter.processViewInfos(ComposeViewAdapter.android.kt:240)
	at androidx.compose.ui.tooling.ComposeViewAdapter.onLayout_Original(ComposeViewAdapter.android.kt:257)
	at androidx.compose.ui.tooling.ComposeViewAdapter.onLayout(ComposeViewAdapter.android.kt)
	at android.view.View.layout_Original(View.java:25592)
	at android.view.View_Delegate.layout(View_Delegate.java:91)
	at android.view.View.layout(View.java:25569)
	at android.view.ViewGroup.layout(ViewGroup.java:6458)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
	at android.view.View.layout_Original(View.java:25592)
	at android.view.View_Delegate.layout(View_Delegate.java:91)
	at android.view.View.layout(View.java:25569)
	at android.view.ViewGroup.layout(ViewGroup.java:6458)
	at android.widget.RelativeLayout.onLayout(RelativeLayout.java:1103)
	at android.view.View.layout_Original(View.java:25592)
	at android.view.View_Delegate.layout(View_Delegate.java:91)
	at android.view.View.layout(View.java:25569)
	at android.view.ViewGroup.layout(ViewGroup.java:6458)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:332)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
	at android.view.View.layout_Original(View.java:25592)
	at android.view.View_Delegate.layout(View_Delegate.java:91)
	at android.view.View.layout(View.java:25569)
	at android.view.ViewGroup.layout(ViewGroup.java:6458)
	at com.android.layoutlib.bridge.impl.RenderSessionImpl.inflate(RenderSessionImpl.java:374)
	at com.android.layoutlib.bridge.Bridge.createSession(Bridge.java:462)
	at com.android.tools.idea.layoutlib.LayoutLibrary.createSession(LayoutLibrary.java:125)
	at com.android.tools.rendering.RenderTask.createRenderSession(RenderTask.java:829)
	at com.android.tools.rendering.RenderTask.lambda$inflate$7(RenderTask.java:977)
	at com.android.tools.rendering.RenderExecutor.runAsyncActionWithTimeout$lambda$12(RenderExecutor.kt:217)
	at com.android.tools.rendering.RenderExecutor$PriorityRunnable.run(RenderExecutor.kt:338)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.lang.ClassNotFoundException: androidx.compose.runtime.tooling.SourceInformationKt
	at com.android.tools.rendering.classloading.loaders.DelegatingClassLoader.loadClassBytes(DelegatingClassLoader.kt:62)
	at com.android.tools.rendering.classloading.loaders.DelegatingClassLoader.findClass(DelegatingClassLoader.kt:84)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	at com.android.tools.rendering.classloading.loaders.DelegatingClassLoader.loadClass(DelegatingClassLoader.kt:70)
	... 64 more
```

--- a/core/droidkaigiui/src/commonMain/kotlin/io/github/droidkaigi/confsched/droidkaigiui/SaveableHelper.kt
+++ b/core/droidkaigiui/src/commonMain/kotlin/io/github/droidkaigi/confsched/droidkaigiui/SaveableHelper.kt
@@ -23,7 +23,7 @@ import androidx.compose.runtime.saveable.rememberSaveable
 fun rememberBooleanSaveable(initial: Boolean = false): MutableState<Boolean> {
     val saver: Saver<MutableState<Boolean>, Boolean> = Saver(
         save = { it.value },
-        restore = { mutableStateOf(it) }
+        restore = { mutableStateOf(it) },
     )
     return rememberSaveable(saver = saver) { mutableStateOf(initial) }
 }
@@ -32,7 +32,7 @@ fun rememberBooleanSaveable(initial: Boolean = false): MutableState<Boolean> {
 fun rememberIntSaveable(initial: Int = 0): MutableState<Int> {
     val saver: Saver<MutableState<Int>, Int> = Saver(
         save = { it.value },
-        restore = { mutableStateOf(it) }
+        restore = { mutableStateOf(it) },
     )
     return rememberSaveable(saver = saver) { mutableStateOf(initial) }
 }

--- a/core/droidkaigiui/src/commonMain/kotlin/io/github/droidkaigi/confsched/droidkaigiui/SaveableHelper.kt
+++ b/core/droidkaigiui/src/commonMain/kotlin/io/github/droidkaigi/confsched/droidkaigiui/SaveableHelper.kt
@@ -1,0 +1,38 @@
+package io.github.droidkaigi.confsched.droidkaigiui
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.Saver
+import androidx.compose.runtime.saveable.rememberSaveable
+
+/**
+ * Why this exists:
+ * - `rememberSaveable<T>` relies on a reified type parameter (T) and must be inlined.
+ * - In some call paths (e.g., context-receiver extension composables, function references,
+ *   mixed AndroidX/JB Compose toolchains, or preview classpaths), the call may not be inlined,
+ *   triggering `UnsupportedOperationException: This function has a reified type parameter...`.
+ * - Providing an explicit `Saver<MutableState<Boolean>, Boolean>` avoids the reified `autoSaver<T>()`
+ *   path entirely, while preserving process-death persistence semantics.
+ * - Prefer this helper over `rememberSaveable { mutableStateOf(false) }` when used inside extension
+ *   composables or if you hit the above crash. If persistence is not required, use `remember { ... }`.
+ * Implementation notes:
+ * - Ensure AndroidX imports (`androidx.compose.runtime.saveable.*`).
+ */
+@Composable
+fun rememberBooleanSaveable(initial: Boolean = false): MutableState<Boolean> {
+    val saver: Saver<MutableState<Boolean>, Boolean> = Saver(
+        save = { it.value },
+        restore = { mutableStateOf(it) }
+    )
+    return rememberSaveable(saver = saver) { mutableStateOf(initial) }
+}
+
+@Composable
+fun rememberIntSaveable(initial: Int = 0): MutableState<Int> {
+    val saver: Saver<MutableState<Int>, Int> = Saver(
+        save = { it.value },
+        restore = { mutableStateOf(it) }
+    )
+    return rememberSaveable(saver = saver) { mutableStateOf(initial) }
+}

--- a/feature/contributors/src/commonMain/kotlin/io/github/droidkaigi/confsched/contributors/component/ContributorsCounter.kt
+++ b/feature/contributors/src/commonMain/kotlin/io/github/droidkaigi/confsched/contributors/component/ContributorsCounter.kt
@@ -15,8 +15,6 @@ import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.LaunchedEffect
 import androidx.compose.runtime.getValue
-import androidx.compose.runtime.mutableIntStateOf
-import androidx.compose.runtime.saveable.rememberSaveable
 import androidx.compose.runtime.setValue
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
@@ -25,6 +23,7 @@ import io.github.droidkaigi.confsched.contributors.ContributorsRes
 import io.github.droidkaigi.confsched.contributors.contributor_person
 import io.github.droidkaigi.confsched.contributors.contributor_total
 import io.github.droidkaigi.confsched.droidkaigiui.KaigiPreviewContainer
+import io.github.droidkaigi.confsched.droidkaigiui.rememberIntSaveable
 import org.jetbrains.compose.resources.stringResource
 import org.jetbrains.compose.ui.tooling.preview.Preview
 
@@ -33,7 +32,7 @@ fun ContributorsCounter(
     totalContributors: Int,
     modifier: Modifier = Modifier,
 ) {
-    var targetValue by rememberSaveable { mutableIntStateOf(0) }
+    var targetValue by rememberIntSaveable(0)
     val animatedTotalContributor by animateIntAsState(
         targetValue = targetValue,
         animationSpec = tween(

--- a/feature/eventmap/src/commonMain/kotlin/io/github/droidkaigi/confsched/eventmap/EventMapScreenPresenter.kt
+++ b/feature/eventmap/src/commonMain/kotlin/io/github/droidkaigi/confsched/eventmap/EventMapScreenPresenter.kt
@@ -1,8 +1,10 @@
 package io.github.droidkaigi.confsched.eventmap
 
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
 import androidx.compose.runtime.getValue
 import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.Saver
 import androidx.compose.runtime.saveable.rememberSaveable
 import androidx.compose.runtime.setValue
 import io.github.droidkaigi.confsched.common.compose.EventEffect
@@ -18,7 +20,11 @@ fun eventMapScreenPresenter(
     events: PersistentList<EventMapEvent>,
     eventFlow: EventFlow<EventMapScreenEvent>,
 ) = providePresenterDefaults {
-    var selectedFloor by rememberSaveable { mutableStateOf(FloorLevel.Ground) }
+    val floorSaver: Saver<MutableState<FloorLevel>, String> = Saver(
+        save = { it.value.name },
+        restore = { mutableStateOf(FloorLevel.valueOf(it)) }
+    )
+    var selectedFloor by rememberSaveable(saver = floorSaver) { mutableStateOf(FloorLevel.Ground) }
 
     EventEffect(eventFlow) { event ->
         when (event) {

--- a/feature/eventmap/src/commonMain/kotlin/io/github/droidkaigi/confsched/eventmap/EventMapScreenPresenter.kt
+++ b/feature/eventmap/src/commonMain/kotlin/io/github/droidkaigi/confsched/eventmap/EventMapScreenPresenter.kt
@@ -22,7 +22,7 @@ fun eventMapScreenPresenter(
 ) = providePresenterDefaults {
     val floorSaver: Saver<MutableState<FloorLevel>, String> = Saver(
         save = { it.value.name },
-        restore = { mutableStateOf(FloorLevel.valueOf(it)) }
+        restore = { mutableStateOf(FloorLevel.valueOf(it)) },
     )
     var selectedFloor by rememberSaveable(saver = floorSaver) { mutableStateOf(FloorLevel.Ground) }
 

--- a/feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/components/TimetableItemDetailContent.kt
+++ b/feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/components/TimetableItemDetailContent.kt
@@ -17,7 +17,6 @@ import androidx.compose.runtime.Composable
 import androidx.compose.runtime.getValue
 import androidx.compose.runtime.mutableStateOf
 import androidx.compose.runtime.remember
-import androidx.compose.runtime.saveable.rememberSaveable
 import androidx.compose.runtime.setValue
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.text.style.TextOverflow
@@ -27,6 +26,7 @@ import io.github.droidkaigi.confsched.designsystem.theme.LocalRoomTheme
 import io.github.droidkaigi.confsched.designsystem.theme.ProvideRoomTheme
 import io.github.droidkaigi.confsched.droidkaigiui.KaigiPreviewContainer
 import io.github.droidkaigi.confsched.droidkaigiui.extension.roomTheme
+import io.github.droidkaigi.confsched.droidkaigiui.rememberBooleanSaveable
 import io.github.droidkaigi.confsched.model.core.Lang
 import io.github.droidkaigi.confsched.model.sessions.TimetableItem
 import io.github.droidkaigi.confsched.model.sessions.fake
@@ -60,7 +60,7 @@ private fun DescriptionSection(
     description: String,
     onLinkClick: (url: String) -> Unit,
 ) {
-    var isExpand by rememberSaveable { mutableStateOf(false) }
+    var isExpand by rememberBooleanSaveable(false)
     var isOverFlow by remember { mutableStateOf(false) }
 
     Column(modifier = Modifier.padding(8.dp)) {

--- a/gradle-conventions/src/main/kotlin/droidkaigi/convention/kmp-feature.gradle.kts
+++ b/gradle-conventions/src/main/kotlin/droidkaigi/convention/kmp-feature.gradle.kts
@@ -43,10 +43,6 @@ kotlin {
             implementation(libs.library("lifecycleRuntimeCompose"))
         }
 
-        androidMain.dependencies {
-            implementation(libs.library("lifecycleViewmodelNavigation3"))
-        }
-
         jvmMain.dependencies {
             implementation(compose.desktop.currentOs)
         }

